.github/release.yml

```diff
@@ -0,0 +1,10 @@
+# .github/release.yml
+
+changelog:
+  exclude:
+    authors:
+      - dependabot
+  categories:
+    - title: release-
+      labels:
+        - '*'
```

Comment:
I'm a bit confused by this. Doesn't this chuck every labelled PR into a category called "release-"? Why that name, and why not include unlabelled PRs?

Reply:
Turns out I am the one who is confused about this. I was thrown off by the `Required` label in the description (GitHub documentation). Have created a separate repo to test - they were indeed not needed. It is only required if I wanted to include categories in the changelog. The documentation could have been a bit clearer, or maybe the user could be a bit better (i.e. me 😅 )

Reply:
Very cool that you're testing in a separate repo!